heron/api/src/java/com/twitter/heron/streamlet/KeyValue.java

```diff
@@ -31,6 +31,10 @@
     return new KeyValue<R, T>(k, v);
   }
 
+  public KeyValue() {
+    // nothing really
```

Comment:
what is this dummy ctor for? Is it better to have mis-usage protection if this is required by kryo?

Reply:
This is primarily for Kryo. Made it non public